pkg/sdk/tasks_impl_gen.go

```diff
@@ -218,6 +274,29 @@ func (r taskDBRow) convert() *Task {
 	return &task
 }
 
+func getPredecessors(predecessors string) ([]string, error) {
```

Comment:
I'm not sure if Trim and LastIndex functions will remove \" every time. I've wrote a simple test for the cases you mentioned and it failed for second case. Take a look

```go
func TestLastIdx(t *testing.T) {
	tcs := []string{
		"\".\"x",
		"\".x",
		"\".\".x",
	}
	for i, tc := range tcs {
		formattedName := strings.Trim(tc, "\\\"")
		idx := strings.LastIndex(formattedName, "\"") + 1
		if strings.LastIndex(formattedName, ".\"")+2 < idx {
			idx++
		}
		require.Equal(t, "x", formattedName[idx:], fmt.Sprintf("failed for %d", i))
	}
}
```

Reply:
This were not the complete cases, just parts of them - how the "last" dot in the identifier is placed, depending on quotations around it (the ones that were failing - it appears that the old implementation was just bad, and still is not perfect).

I will just remove the comment because all of these three options are provided in the unit test and they are passing.